cmd/bro-backend/commands/root.go

```diff
@@ -4,6 +4,6 @@ import "github.com/spf13/cobra"
 
 func Execute() {
 	var rootCmd = &cobra.Command{Use: "bro-backend"}
-	rootCmd.AddCommand(getSpinCommand())
+	rootCmd.AddCommand(getSpinCommand(), getStatusCommand())
```

Comment:
i suggest to not add status command in the first place if the OS is not Linux. Instead of making it crashed.

Reply:
oh so, no linux no bro?